book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,555 @@
+- [ ] Decide on assessments and criteria
+    * Be clear on what criteria is being used for shortlisting
+    * Ensure assessment criteria focus on the person specification within the job description. This should help mitigate bias during hiring decisions later on.
+
+
+### At the deadline 
+
+
+
+- [ ] Close job advert 
+    * Be transparent with the status of the application and timelines
+    * Ensure reasonable timeframes are kept when making decisions 
+    * Either at this stage, or at the end of the process, review anonymised demographic data of candidates to see if your advertising avenues were ‘effective’, how ever this may be determined by your organisation
+
+
+### 
+
+
+## Resources
+
+
+### Further reading 
+
+Included below are a small selection of resources for further reading, that can expand your knowledge on diverse recruitment practices. 
+This is not a comprehensive list and any organisations, groups or documents listed should be reviewed by an individual before engaging with them. 
+
+Recruiting a diverse technical workforce checklist - [https://www.stemm-change.co.uk/wp-content/uploads/2021/09/Recruiting-a-diverse-technical-workforce-smaller.pdf](https://www.stemm-change.co.uk/wp-content/uploads/2021/09/Recruiting-a-diverse-technical-workforce-smaller.pdf)
+
+EDI recruitment checklist - [https://n8cir.org.uk/documents/140/2022_05_09_recrtuitment_checklist_K3EWMHP.pdf](https://n8cir.org.uk/documents/140/2022_05_09_recrtuitment_checklist_K3EWMHP.pdf)
+
+Diversity and inclusion checklist for recruitment, improving workforce diversity and other resources - [https://womeninhpc.org/community/resources#!/women-resources](https://womeninhpc.org/community/resources#!/women-resources) 
+
+Inclusive recruitment: Guide for people professionals - [https://www.cipd.org/uk/knowledge/guides/inclusive-employers/](https://www.cipd.org/uk/knowledge/guides/inclusive-employers/)
+
+How employers should recruit - [https://www.acas.org.uk/hiring-someone](https://www.acas.org.uk/hiring-someone) 
+
+Providing salary transparency in job adverts - [https://www.vercidagroup.com/post/show-them-the-money-how-including-salary-in-job-ads-can-help-build-a-more-diverse-workforce](https://www.vercidagroup.com/post/show-them-the-money-how-including-salary-in-job-ads-can-help-build-a-more-diverse-workforce)
+
+Digital accessibility on social media - https://book.the-turing-way.org/community-handbook/accessibility/accessibility-comms-platforms
+
+Alternative Text - https://book.the-turing-way.org/community-handbook/accessibility/accessibility-alt-text
+
+### Useful organisations for data science & STEM hiring in the UK 
+
+The below resources are a selection of data science, AI and tech spaces which may be useful for posting your job advert or engaging with specific groups/organisations. 
+The target audience for these groups has been identified from each of their webpages. 
+They mostly have a UK focus, though some are more international. 
+
+
+<table>
```

Comment:
We can double check this but I don't think we can use HTML formatting since this is a markdown file